substrate/frame/src/lib.rs

```diff
@@ -202,12 +202,20 @@ pub mod prelude {
 	/// Dispatch types from `frame-support`, other fundamental traits
 	#[doc(no_inline)]
 	pub use frame_support::dispatch::{GetDispatchInfo, PostDispatchInfo};
-	pub use frame_support::traits::{
-		Contains, EitherOf, EstimateNextSessionRotation, Everything, IsSubType, MapSuccess,
-		NoOpPoll, OnRuntimeUpgrade, OneSessionHandler, RankedMembers, RankedMembersSwapHandler,
-		VariantCount, VariantCountOf,
+	pub use frame_support::{
+		defensive, defensive_assert, ensure,
+		traits::{
+			fungible::{Inspect, InspectFreeze, Mutate, MutateFreeze},
```

Comment:
I don't think we should import the fungible trait like this. they will collide with `fungibles` one.